### PR TITLE
[OSDOCS-3688]: 4.11 Rel Notes for Cloud team features

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.11

Issue:
[OSDOCS-3688](https://issues.redhat.com//browse/OSDOCS-3688) and subtasks ([OSDOCS-3690](https://issues.redhat.com//browse/OSDOCS-3690) [OSDOCS-3837](https://issues.redhat.com//browse/OSDOCS-3837) [OSDOCS-3827](https://issues.redhat.com//browse/OSDOCS-3827) [OSDOCS-3834](https://issues.redhat.com//browse/OSDOCS-3834) [OSDOCS-3838](https://issues.redhat.com//browse/OSDOCS-3838))

Link to docs preview:
- [New features and enhancements > Machine API](http://file.rdu.redhat.com/jrouth/OSDOCS-3688-cloud-rel-notes/release_notes/ocp-4-11-release-notes.html#ocp-4-11-machine-api)
- [Technology Preview features](http://file.rdu.redhat.com/jrouth/OSDOCS-3688-cloud-rel-notes/release_notes/ocp-4-11-release-notes.html#ocp-4-11-technology-preview) (end of table)

Additional information:
~The CAPI TP docs are still in progress, so the link from *Managing machines with the Cluster API (Technology Preview)* is broken (and I expect the build to fail) until #48121 is merged)~
CAPI TP docs are merged, and the build is resolved :tada:

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@sunzhaohua2 this should resolve your [RN tracker comment](https://github.com/openshift/openshift-docs/issues/43249#issuecomment-1157342886), PTAL

Note: AWS EFA is bumped to 4.12 due to some dependency complications.

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -587,6 +587,41 @@ For more information, see xref:../operators/operator_sdk/java/osdk-java-quicksta
 [id="ocp-4-11-machine-api"]
 === Machine API
 
+[id="ocp-4-11-mapi-aws-imdsv2"]
+==== Configuration options for the Amazon EC2 Instance Metadata Service
+
+You can now use machine sets to create compute machines that use a specific version of the Amazon EC2 Instance Metadata Service (IMDS). Machine sets can create compute machines that allow the use of both IMDSv1 and IMDSv2 or compute machines that require the use of IMDSv2.
```
Comment (Amrita42) [inline release_notes/ocp-4-11-release-notes.adoc:593]: Suggestion >> Can this be phrased as **Machine sets can create compute machines that allow the use of both IMDSv1 and IMDSv2 or just IMDSv2**

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -587,6 +587,41 @@ For more information, see xref:../operators/operator_sdk/java/osdk-java-quicksta
...
+
+For more information, see xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#machineset-imds-options_creating-machineset-aws[Machine set options for the Amazon EC2 Instance Metadata Service].
+
+[id="ocp-4-11-mapi-ultra-disks"]
+==== Machine API support for Azure ultra disks
+
+You can now create a machine set running on Azure that deploys machines with ultra disks. You can either deploy machines with ultra disks as data disks, or deploy machines with ultra disks by using persistent volume claims (PVCs) that use in-tree or Container Storage Interface (CSI) PVCs.
```
Comment (Amrita42) [inline release_notes/ocp-4-11-release-notes.adoc:600]: Suggestion>> _You can either deploy machines with ultra disks as data disks, or by using persistent volume claims (PVCs) that use in-tree or Container Storage Interface (CSI) PVCs._

**Pull request merged**